### ISSUE TITLE
Monokle CLI support for cmd and powershell.

### ISSUE THREAD

**Issue by erdkse** [kubeshop/monokle#290]
**Is your feature request related to a problem? Please describe.**
I want the application to also support opening specified directory by CLI command on windows just like MacOs. 

**Describe the solution you'd like**
Currently, Windows cmd and sh codes exist in the codebase. I would like to add an executable path or symlink to these commands for windows paths. So windows will recognize commands. It needs to be done programmatically while the application is installing on the OS.

**Describe alternatives you've considered**
Gulp installation may be used as VSCode does.

**Additional context**
-

**Comment by olensmar** [comment]
thanks @erdkse - is it possible to configure this manually today using the provided cmd/sh scripts? If so perhaps we can add corresponding instructions to the docs?

**Comment by erdkse** [comment]
Yes, It is possible. But It may differ from installation to installation. Because the paths are relevant. I will write instructions for it.
